src/main/java/net/atos/zac/app/zaken/ZakenRESTService.java

```diff
@@ -534,8 +535,8 @@ public void afbreken(@PathParam("uuid") final UUID zaakUUID, final RESTZaakAfbre
                 zrcClientService.readStatus(zaak.getStatus()).getStatustype()) : null;
         assertPolicy(
                 zaak.isOpen() && !isHeropend(statustype) && policyService.readZaakRechten(zaak).getAfbreken() && !zrcClientService.heeftOpenDeelzaken(zaak));
-        final ZaakbeeindigParameter zaakbeeindigParameter = zaakafhandelParameterBeheerService.readZaakbeeindigParameter(
-                uuidFromURI(zaak.getZaaktype()), afbrekenGegevens.zaakbeeindigRedenId);
+        final ZaakafhandelParameters zaakafhandelParameters = zaakafhandelParameterService.readZaakafhandelParameters(UriUtil.uuidFromURI(zaak.getZaaktype()));
```

Comment:
Zaaktype heeft een convenience method: getUUID() Dan is die UriUtil.uuidFromURI(zaak.getZaaktype()) niet nodig.